utils.d.ts

```diff
@@ -0,0 +1,38 @@
+type ThirdPartyExtensionSubject = {
+  name: string;
+  url: string;
+};
+
+export type ThirdPartyExtension = {
+  module: ThirdPartyExtensionSubject;
+  author: ThirdPartyExtensionSubject;
+};
+
+export type IconData = {
+  title: string;
+  hex: string;
+  source: string;
+  slug: string | undefined;
+  guidelines: string | undefined;
+  license: string | any | undefined;
+  aliases: any | undefined;
+};
```

Comment:
We should reuse the `SimpleIcon` type.

```js
export function getIconSlug(icon: SimpleIcon): string;
```

Reply:
Nor `svg` nor `path` properties are part of icon data objects.